package.json

```diff
@@ -436,6 +436,15 @@
             "type": "boolean",
             "default": false
           },
+          "terraform.experimentalFeatures.languageServer.tcp.port": {
```

Comment:
Is there a particular reason you want to roll this out as experimental? At least the code change looks relatively straight-forward and I wouldn't expect many people actually end up using it for day2day development anyway, so there's already reduced risk/exposure from that angle.

Reply:
I was were trying to imply this wasn't a primary workflow we supported for every day use. If we think it's fine alongside the primary workflow I'm ok with removing the experimental part